doc/source/user_guide/scale.rst

```diff
@@ -217,190 +217,10 @@ require too sophisticated of operations. Some operations, like :meth:`pandas.Dat
 much harder to do chunkwise. In these cases, you may be better switching to a
 different library that implements these out-of-core algorithms for you.
 
-.. _scale.other_libraries:
```

Comment:
Can you leave this label please, so we can link to this section if needed.

Reply:
I see we have [warning undefined label](https://github.com/pandas-dev/pandas/actions/runs/8287460372/job/22679861006?pr=57843#step:7:64) because I removed it. Makes sense. Thanks for letting me know.